staging/src/k8s.io/apimachinery/pkg/util/httpstream/wsstream/conn.go

```diff
@@ -178,7 +179,11 @@ func (conn *Conn) Open(w http.ResponseWriter, req *http.Request) (string, []io.R
 	go func() {
 		defer runtime.HandleCrash()
 		defer conn.Close()
-		websocket.Server{Handshake: conn.handshake, Handler: conn.handle}.ServeHTTP(w, req)
+		handler := conn.handle
+		if _, ok := conn.protocols[remotecommand.StreamProtocolV5Name]; ok {
+			handler = conn.handleV5
+		}
```

Comment:
I do not know if we decided to use a featuregate here

Reply:
The only way the websocket server negotiated subprotocol version is V5 is when this code is included in the `StreamTranslator`. The `StreamTranslator` code is protected by the feature gate. Also, I believe there are dependency issues with referencing the feature gate here in `apimachinery` (since the feature gate is in `apiserver`).

Reply:
fair enough